core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +56,13 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
```

Comment:
Thanks for the clarification.To reserve some space, there is two methods, use fixed few GB, or use percentage of the worker's capacity, which do you think is better? If use fixed GB, how many will be more suitable?